common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -759,8 +739,6 @@
                     self.rendered = true;
                     self.state.el.addClass('is-captions-rendered');
 
-                    self.showTemporaryCaptionMessage(self, "<li>Transcript will be displayed when you start playing the video.</li>"); // jshint ignore: line
```

Comment:
I created that function originally to run through the HtmlUtil stuff in case these messages were used more than once. Turns out this is the only place so I simplified it and did it inline, removing the function since it wasn't being used.